bittytax/conv/parsers/binance.py

```diff
@@ -342,7 +345,7 @@ def get_buy_quantity(op_rows):
 
 DataParser(DataParser.TYPE_EXCHANGE,
            "Binance Deposits/Withdrawals",
-           ['Date(UTC)', 'Coin', 'Amount', 'Status', 'Payment Method', 'Indicated Amount', 'Fee',
+           ['Date(UTCnull)', 'Coin', 'Amount', 'Status', 'Payment Method', 'Indicated Amount', 'Fee',
```

Comment:
`Date(UTCnull)` this looks like a bug with the Binance export adding a 'null' to the end. Please can you re-export and see if the problem still exists, if so please raise a ticket with Binance.